### DISCUSSION TITLE
# Frequency reverse - whatever that means, don't see it in the manual

### DISCUSSION THREAD

**Discussion by BruMcD** [egzumer/uvk5-chirp-driver#41] [General]
FYI

Noticed the above comment in the Chirp Driver code (Not sure who wrote it).       Quoted /Involved in Revision 91c78b39

**Frequency reverse** asks a radio to consider if a frequency/channel setting  involves a duplex frequency pair being set (e.g for a repeater where transmit might be 0.6MHz lower than the receive or whatever is required) .

If such a pair exists the Frequency reverse function asks the radio to use the 'Receive column' value for Transmit and the 'Transmit column' value for Receive.

It is more commonly known/used as **reverse repeater** as one press would allow you to listen for a very close station's transmission on the **input** of a repeater and (somewhat rudely) shout (go to channel xx simplex instead) **over the repeater's output signal**.

This works fine on the radios,  The Chirp setting allows each channel to have a default on/off status set.

**Comment by egzumer** [comment]
It is a note from Dan the chirp creator. He added it while adapting the chirp driver.

https://github.com/egzumer/uv-k5-firmware-custom/wiki/Button-functions#8-r
